arrow/parser.py

```diff
@@ -351,33 +365,31 @@ def _build_datetime(parts):
 
         timestamp = parts.get("timestamp")
 
-        if timestamp:
+        if timestamp is not None:
```

Comment:
any reason for changing this?

Reply:
I changed it because `parts.get("timestamp")` will yield `None` if `timestamp` is not in the parts dictionary, and we only want to account for `None` cases here. It is best to be as explicit as possible :).

`if timestamp` handles more cases than just `None` cases (check out [this StackOverflow post](https://stackoverflow.com/questions/3901144/difference-between-if-x-and-if-x-is-not-none)).